molecule/default/tasks/cluster_info.yml

```diff
@@ -9,7 +9,7 @@
 
 - name: Get core API version
   set_fact:
-    crd: "{{ api_details.apis['apiextensions.k8s.io'] }}"
+    crd: "{{ api_details.apis['apiextensions.k8s.io/v1'] }}"
```

Comment:
I think we should probably add some checks to make sure this is populating the resources in each group, so pick a group that has multiple resources like `apps/v1/DaemonSet` and `apps/v1/Deployment`.